config.py

```diff
@@ -133,6 +133,18 @@ class TestingConfig(DevelopmentConfig):
     WTF_CSRF_ENABLED = False
     SESSION_TYPE = "filesystem"
     SESSION_PERMANENT = False
-    UAA_PUBLIC_KEY = 'Test'
+    UAA_PUBLIC_KEY = '''-----BEGIN PUBLIC KEY-----
```

Comment:
Requesting approval of IA

Reply:
I think it's likely fine, but depends where this test key came from.  So long as it's not reused anywhere, or just copied from someone's `~/.ssh/` dir, should be fine.

Reply:
The key was generated using http://travistidwell.com/jsencrypt/demo/ and isn't used anywhere else.  It's only used for the tests and isn't used anywhere else.
I needed it so I could emulate how the key would be encoded when the code is running for real.

Reply:
I'd be tempted to either comment to mention that it's a demo key, or to create a zeroed key to make it obvious.